CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# [](https://github.com/algolia/autocomplete.js/compare/v1.0.0-alpha.30...v) (2020-09-28)
```

Comment:
current version number seems missing here (in ...v, as well as the text of the link

Reply:
Yeah that's a Ship.js bug–I'm adding it manually every time before merging the PR.